README.md

```diff
@@ -59,7 +59,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+The value of these expressions is that weather equals the string "sunny". The = means it equals "sunny" but === means it must have the equal value of "sunny".
```

Comment:
This is correct. However, the question was asking what each line evaluated to. The first line evaluates to `undefined`; the second line evaluates to `"sunny"`, because the `=` operator evaluates to the value that it is assigning; the third line, evaluates to `true`.